Makefile

```diff
@@ -14,10 +14,17 @@ ifeq (${CI}, true)
 	endif
 endif
 
+SANDBOX_TESTING = sandbox-cgroup
+ifeq (${CI}, true)
+        ifneq (${TEST_SANDBOX}, true)
```

Comment:
You are going to create a new job that exports this variable? Maybe TEST_SANDBOX_CGROUP_ONLY may be more explicit